language/e2e-tests/src/common_transactions.rs

```diff
@@ -260,6 +246,38 @@ pub fn set_validator_config_txn(
     )
 }
 
+/// Returns a transaction to set config for a candidate validator
+pub fn set_validator_config_and_reconfigure_txn(
```

Comment:
is the set_validator_config txn only used in genesis build after this?

Reply:
should we be using the transaction builder? (see below)

Reply:
@zekun000 , not just in genesis, but any time we want to add a new validator: the validator will set its config locally first (without reconfig) before being added to the validator set.

Reply:
ah, can we rename that to something like `register_validator_config` and make it clear it publishes new config instead of changing it.

Reply:
Sure, that's a good idea!

Reply:
This PR becomes a little crowded, I'll put a TODO and rename in the next PR as all the accompanying rust functions will also have to be renamed as well...